fearless/Common/Services/RemoteSubscription/WalletRemoteSubscriptionService.swift

```diff
@@ -20,26 +19,29 @@ protocol WalletRemoteSubscriptionServiceProtocol {
 class WalletRemoteSubscriptionService: RemoteSubscriptionService, WalletRemoteSubscriptionServiceProtocol {
     func attachToAccountInfo(
         of accountId: AccountId,
-        chain: ChainModel,
+        chainAsset: ChainAsset,
         queue: DispatchQueue?,
         closure: RemoteSubscriptionClosure?
     ) -> UUID? {
         do {
-            let storagePath = chain.isOrml ? StorageCodingPath.tokens : StorageCodingPath.account
+            var storagePath: StorageCodingPath
+            switch chainAsset.chainAssetType {
```

Comment:
it will be move universal if you move this logic to ChainAsset extension (e.g. ChainAsset+Palletes). all necessary data stored inside ChainAsset. this way you can avoid code duplicates in case you need to decide what pallete to use in other modules.